frameworks/elastic/docs/install.md

```diff
@@ -113,6 +113,31 @@ Elastic supports deployment on virtual networks on DC/OS (including the `dcos` o
 ```
 As mentioned in the [developer guide](https://mesosphere.github.io/dcos-commons/developer-guide.html) once the service is deployed on a virtual network, it cannot be updated to use the host network.
 
+## Zones
+
+Placement constraints can be applied to zones by referring to the `@zone` key. For example, one could spread pods across a minimum of 3 different zones by specifying the constraint `@zone:GROUP_BY:3`.
+ 
+<!--
+When the region awareness feature is enabled (currently in beta), the `@region` key can also be referenced for defining placement constraints. Any placement constraints that do not reference the `@region` key are constrained to the local region.
+-->
+## Example
+
+Suppose we have a Mesos cluster with zones `a`,`b`,`c`.
+
+## Balanced Placement for a Single Region
+
+```
+{
+   ...
+  "instances": 6,
+  "constraints": [
+    ["@zone", "GROUP_BY", "3"]
```

Comment:
@mpereira What's the right syntax here, for the new JSON format we're using.

Reply:
It's an array of constraint-arrays like in the current form but it should be stringified. So:
```
"constraints": "[[\"@zone\", \"GROUP_BY\", \"3\"]]"
```

I don't know if it matters here but `placement` values for Elasticsearch should be nested under `master_nodes`, `data_nodes`, `ingest_nodes` or `coordinator_nodes`, e.g.:
```
{
  ...,
  "master_nodes": {
    ...,
    "constraints": "[[\"@zone\", \"GROUP_BY\", \"3\"]]",
    ...
  },
  ...
}
```